packages/react-pdf/src/renderer.js

```diff
@@ -89,24 +92,28 @@ const PDFRenderer = ReactFiberReconciler({
     text,
     rootContainerInstance,
     hostContext,
-    internalInstanceHandle,
+    internalInstanceHandle
   ) {
-    return {
-      text,
-      tag: 'TEXT',
-    };
+    // return text;
+    return createElement('TEXT', { content: 'TEXT' });
+    // return {
+    //   text,
+    //   tag: 'TEXT',
+    // };
   },
 
   commitTextUpdate(textInstance, oldText, newText) {
     textInstance.text = newText;
   },
 
   appendChild(parentInstance, child) {
-    const index = parentInstance.children.indexOf(child);
-    if (index !== -1) {
-      parentInstance.children.splice(index, 1);
-    }
-    parentInstance.children.push(child);
+    child.inject(parentInstance);
```

Comment:
Shouldn't be adding the child to the parentInstance, and not the opposite as we are doing? I know it may seem the same. but it's a bit confusing. Would you mind explaining to me?